front/app/components/admin/PostManager/components/FilterSidebar/index.tsx

```diff
@@ -198,28 +189,60 @@ const FilterSidebar = ({
   const selectedItem = items.find((i) => i.key === activeFilterMenu);
   return (
     <>
-      <Menu
-        tabular
-        attached="top"
-        size="tiny"
+      <Box
+        display="flex"
+        flexDirection="row"
         className="intercom-admin-input-manager-filter-sidebar"
       >
-        {items.map((item) => (
-          <Menu.Item
-            key={item.key}
-            id={item.key}
-            active={activeFilterMenu === item.key}
-            onClick={handleItemClick}
-            className={`intercom-admin-input-manager-filter-sidebar-${item.key}`}
-            data-cy={`e2e-admin-post-manager-filter-sidebar-${item.key}`}
-          >
-            {item.name}
-          </Menu.Item>
-        ))}
-      </Menu>
-      <Segment attached="bottom">{selectedItem.content}</Segment>
+        {items.map((item) => {
+          const active = activeFilterMenu === item.key;
+
+          return (
+            <Box
+              display="flex"
+              flexDirection="row"
+              justifyContent="space-between"
```

Comment:
```suggestion
              justifyContent="space-between"
              alignItems="center
```
This one is missing.